components/__init__.py

```diff
@@ -1,14 +1,14 @@
 from .component import Component
 from .data_field import DataField, SpeedField
 from .centre_power import CentrePower
-from .message import Message
+from .dashboard_message import DAShboardMessage
 from .das_disconnect_message import DASDisconnectMessage
 
 __all__ = [
     "Component",
     "DataField",
     "SpeedField",
     "CentrePower",
-    "Message",
+    "DAShboardMessage",
```

Comment:
Include `Message` here as well! Will help below

Reply:
Is this file required when importing classes from a folder?

Reply:
Not required, but it can definitely make things neater! This defines what the module `components` will contain, meaning we can do things like `from components import DAShboardMessage` rather than needing to specify the exact file too with `from components.dashboard_message import DAShboardMessage`. It also defines what will be imported with `from components import *`, although this is generally not something we do.